sdk-go/littlehorse/wf_lib_public.go

```diff
@@ -93,6 +105,10 @@ func (n *ExternalEventNodeOutput) JsonPath(jsonPath string) NodeOutput {
 	}
 }
 
+func (n *ExternalEventNodeOutput) RegisteredAs(payloadType lhproto.VariableType) *ExternalEventNodeOutput {
```

Comment:
Same here, this should be a `TypeDefinition` not a `VariableType`.

@Snarr would you care to chime in about how this might work with `StructDef` registration?